### PR TITLE
fix: fixed the non-effective plugin configuration update in tcp/udp-logger plugin

### PR DESCRIPTION
**Pull request by tokers**
### What this PR does / why we need it:

Closing https://github.com/apache/apisix/issues/2922.
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: apisix/plugins/tcp-logger.lua [hunk 1]
```diff
@@ -106,7 +108,7 @@ local function remove_stale_objects(premature)
 
     for key, batch in ipairs(buffers) do
         if #batch.entry_buffer.entries == 0 and #batch.batch_to_process == 0 then
-            core.log.debug("removing batch processor stale object, route id:", tostring(key))
+            core.log.warn("removing batch processor stale object, route id:", tostring(key))
```
Comment (spacewander) [inline apisix/plugins/tcp-logger.lua:111]: The "key" is no longer route id?

Comment (tokers) [reply apisix/plugins/tcp-logger.lua]: @spacewander Nice capture, already fixed.

File: apisix/plugins/tcp-logger.lua [hunk 2]
```diff
@@ -106,7 +108,8 @@ local function remove_stale_objects(premature)
 
     for key, batch in ipairs(buffers) do
         if #batch.entry_buffer.entries == 0 and #batch.batch_to_process == 0 then
-            core.log.debug("removing batch processor stale object, route id:", tostring(key))
+            core.log.warn("removing batch processor stale object, conf: ",
+                          core.json.delay_encode(key))
```
Comment (spacewander) [inline apisix/plugins/tcp-logger.lua:112]: Should we also update? https://github.com/apache/apisix/blob/a52b5e9b3b00da40705144c868cfe3a67de47b8d/apisix/plugins/http-logger.lua#L183

Comment (tokers) [reply apisix/plugins/tcp-logger.lua:112]: Yes.

Comment (tokers) [reply apisix/plugins/tcp-logger.lua:112]: @spacewander Fixed, also fixed it in kakfa-logger.lua

**Review by spacewander — APPROVED** [review]

**Review by membphis — APPROVED** [review]

**Pull request merged**